src/System.Runtime/tests/System/Reflection/CustomAttributeDataTests.cs

```diff
@@ -136,7 +136,7 @@ public static void Test_CustomAttributeData_ToString()
             {
                 if (cad.AttributeType == typeof(MyAttribute))
                 {
-                    Assert.Equal("[System.Reflection.Tests.CustomAttributeDataTests+MyAttribute((Int32)3)]", cad.ToString());
```

Comment:
It sends the wrong message that we're guaranteeing the contents of ToString() on this type.